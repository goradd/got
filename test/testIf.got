package testOut


func TestIf(buf *bytes.Buffer) {
    a:= 1
    b := 1

{{if a == b }}
{{
    Hi there.
}}
{{if}}

}