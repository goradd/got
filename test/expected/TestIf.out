    Hi there.
    Bye there.
    Be square.
